soap_api/mastr_wind_process.py

```diff
@@ -33,15 +34,19 @@ def make_wind(eeg=False):
     if eeg==True:
         unit_wind_eeg = read_unit_wind_eeg(fname_wind_eeg)
     unit_wind_permit = read_unit_wind_permit(fname_wind_permit)
-
+    log.info("Joining tables...")
     table_wind = power_unit_wind \
+    .merge(unit_wind_permit.set_index('EinheitMastrNummer'), on= ['Einheitart', 'Einheittyp', 'GenMastrNummer','EinheitMastrNummer'], how='left') \
+    .merge(unit_wind_eeg.set_index('EegMastrNummer'), on=['EegMastrNummer'], how='left') \
+
+    """table_wind = power_unit_wind \
```

Comment:
is this supposed to be a block comment ?

Reply:
fixed it, typo :)